docs/api-inference/tasks/question_answering.md

```diff
@@ -0,0 +1,127 @@
+## Question-answering
```

Comment:
Do we want to use the normal names rather than pipeline?

```suggestion
## Question Answering
```

Reply:
https://github.com/huggingface/hub-docs/pull/1394/commits/f0785b860b7f5b243ffd93fde92675e7a1ecfd97